bin/lib/check_reqs.js

```diff
@@ -196,20 +208,22 @@ module.exports.check_android = function() {
                 process.env['ANDROID_HOME'] = grandParentDir;
                 hasAndroidHome = true;
             } else {
-                throw new Error('ANDROID_HOME is not set and no "tools" directory found at ' + parentDir);
+                throw new Error('Failed to find \'ANDROID_HOME\' environment variable. Try setting setting it manually.\n' +
+                    'Detected \'android\' command at ' + parentDir + ' but no \'tools\' directory found near.\n' +
+                    'Try reinstall Android SDK or update your PATH to include path to valid SDK directory.');
             }
         }
         if (hasAndroidHome && !adbInPath) {
             process.env['PATH'] += path.delimiter + path.join(process.env['ANDROID_HOME'], 'platform-tools');
         }
         if (!process.env['ANDROID_HOME']) {
-            throw new Error('ANDROID_HOME is not set and "android" command not in your PATH. You must fulfill at least one of these conditions.');
+            throw new Error('Failed to find \'ANDROID_HOME\' environment variable. Try setting setting it manually.\n' +
+                'Failed to find \'android\' command in your \'PATH\'. Try update your \'PATH\' to include path to valid SDK directory.');
         }
         if (!fs.existsSync(process.env['ANDROID_HOME'])) {
-            throw new Error('ANDROID_HOME is set to a non-existant path: ' + process.env['ANDROID_HOME']);
+            throw new Error('\'ANDROID_HOME\' environment variable is leads to non-existant path: ' + process.env['ANDROID_HOME'] +
```

Comment:
"is leads to" isn't good grammar :smile: ... Either leave it as it was ('is set to'), or just 'leads to' (I actually think it worked fine how it was).